app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly ensure that you enabled `:confirmable` in the user model which should allow a sure to confirm the registration process below proceeding. You should see something similar to what is below: :point_down: 
  
  ```ruby
	  devise :database_authenticatable, :registerable,
				  :recoverable, :rememberable, :validatable, :confirmable
  ```